### PR TITLE
docs: include Vuetify SASS variables overriding

### PR DESCRIPTION
**Pull request by anonmos**
### Description

Overriding SASS variables was difficult to figure out when using this module.  Provide some sample documentation for getting it to work.

### Linked Issues

N/A

### Additional Context

Out of the box the documentation on the Vuetify site does not provide a way to get a working example up and running for overriding SASS variables that is compatible with the Vuetify Nuxt Module.  Additionally there is no existing documentation on this module's page for getting it to work.

This extra md file provides step-by-step instructions for configuring SASS variable overrides that work with this Nuxt module.

---

> [!TIP]
> The author of this PR can publish a _preview release_ by commenting `/publish` below.

### CODE REVIEW

**Comment by userquin** [comment]
Move the entry using h2 (`##`) to https://nuxt.vuetifyjs.com/guide/sass-modern-compiler.html, add also a reference somewhere here https://nuxt.vuetifyjs.com/guide/server-side-rendering.html#vuetify-sass-variables to the new h2.

Myabe we can change the menu title here https://nuxt.vuetifyjs.com/guide/sass-modern-compiler.html to `SASS Modern Compiler and SASS Variables`

File: docs/guide/saas-variables.md
Comment (userquin) [inline docs/guide/saas-variables.md:60]: typo here, it is `SASS` not `SAAS`

File: docs/guide/saas-variables.md
Comment (userquin) [inline docs/guide/saas-variables.md:36]: remove trailing `/* ... */` and replace `/* ... */` with `// other options`

File: docs/guide/saas-variables.md
Comment (userquin) [inline docs/guide/saas-variables.md:3]: typo it is `SASS` not `SAAS`

File: docs/guide/saas-variables.md
Comment (userquin) [inline docs/guide/saas-variables.md:56]: remove this and apply previous suggestion

**Comment by anonmos** [comment]
@userquin thanks for the speedy review!  Happy to make more changes as you or @TechAkayy see fit.

File: docs/guide/server-side-rendering.md
Comment (userquin) [inline docs/guide/server-side-rendering.md:53]: replace the link removing the url: `[SASS Variables](/guide/sass-modern-compiler.md#overriding-sass-variables)`.

**Comment by userquin** [comment]
Change VP config file entry here (set text to `SASS Modern Compiler and SASS Variables`): https://github.com/vuetifyjs/nuxt-module/blob/main/docs/.vitepress/config.ts#L194

File: docs/guide/server-side-rendering.md
Comment (userquin) [inline docs/guide/server-side-rendering.md:53]: `[Overriding SASS Variables](/guide/sass-modern-compiler.md#overriding-sass-variables).`

File: docs/guide/sass-modern-compiler.md
Comment (userquin) [inline docs/guide/sass-modern-compiler.md:48]: replace this with:
```md
> [!TIP]
> This will import `main.scss` into ALL of your components by default. If you would like different behavior, you can create a different file with these overrides and import that instead.
```

File: docs/guide/sass-modern-compiler.md
Comment (TechAkayy) [inline docs/guide/sass-modern-compiler.md:43]: Sass override files - it's best to specify the path as `@/assets/styles/main.scss` so that it works for both old way & the new nuxt 4 way (`app` folder).

File: docs/guide/sass-modern-compiler.md
Comment (TechAkayy) [inline docs/guide/sass-modern-compiler.md:49]: Pardon my ignorance, I don't under this tip. Is it referring to the fact that the Global Variables override will "apply" (instead of "import") globally to all components? It might help to add an example of the different behaviour.

File: docs/guide/sass-modern-compiler.md
Comment (TechAkayy) [inline docs/guide/sass-modern-compiler.md:16]: As this module is nuxt specific, instead of `${workspace}`, can I suggest using `${srcDir}` as it will cover the old way, and Nuxt 4 new way (`app` folder).

File: docs/guide/sass-modern-compiler.md
Comment (TechAkayy) [inline docs/guide/sass-modern-compiler.md:51]: We could reword by referring to the module option instead. For eg, 

Again in your `nuxt.config.ts`, under the Vuetity module options, disable the Vuetify Styles import for components and instead import the `settings.scss` override file.

**Review by TechAkayy — CHANGES_REQUESTED** [review]
Thanks bunch @anonmos & @userquin. I have left few minor suggestions for your consideration. 

Can I also suggest renaming this page to `Sass customization` as it will look better (instead of the current longer one) on the vertical left menu. 

@userquin, it would be great to have a reference to this page in the Vuetify sass variables page https://vuetifyjs.com/en/features/sass-variables/#sass-variables. 

It could be a simple `info` box like this: 

::: info

If using `nuxt-vuetify-module` in your Nuxt project, refer to the module's documentation for specific instructions on how sass variable override files are imported in `nuxt.config.ts`.

:::

File: docs/guide/sass-modern-compiler.md
Comment (anonmos) [reply docs/guide/sass-modern-compiler.md:36]: @TechAkayy as a first-exposure to the Sass Variables section of the features page, I found it a bit overwhelming.  Seeing it broken down on the component API pages gives me a contextual background for the variables I'm overriding (e.g. [$button-text-transform](https://vuetifyjs.com/en/api/v-btn/#sass-button-text-transform) in the v-btn API page pertains specifically to the v-btn, and by visiting that page directly I can see the full list there of all of the control I have over the button itself).

My vote -- both pages have their place, let's add them both!  I've updated the comment to include a link to the Variable API section.

File: docs/guide/sass-customization.md
Comment (TechAkayy) [inline docs/guide/sass-customization.md:70]: We can override not just the _variables.scss, but all other variables that are imported into `https://github.com/vuetifyjs/vuetify/blob/master/packages/vuetify/src/styles/settings/_index.sass`. So, we could change the url to this file instead of the _variables.scss.

File: docs/guide/sass-customization.md
Comment (TechAkayy) [inline docs/guide/sass-customization.md:52]: Please change settings.scss to `settings.scss` (code).

**Review by TechAkayy — CHANGES_REQUESTED** [review]
Just two more minor feedbacks. Otherwise, it reads really good. Anyone using our module will find this very valuable. Thanks bunch @anonmos 👍 

By the way, renaming `main.scss` to `global.scss` is really a good idea, I will update my sample repos too. On the same line, wondering if renaming `settings.scss` to `components.scss` makes sense, thoughts @anonmos? @userquin?

File: docs/guide/sass-customization.md
Comment (anonmos) [reply docs/guide/sass-customization.md]: @TechAkayy my original thinking on this was to [follow the Vuetify documentation and point directly to the _variables.scss file](https://github.com/vuetifyjs/vuetify/blob/master/packages/docs/src/pages/en/features/sass-variables.md?plain=1#L75).  Additionally, looking at the `_index.sass` file, it feels a bit abstract to point users there, particularly since they'll be viewing it directly through a web browser and would have to subsequently click around on the left-hand file explorer to see any more detail than the highest level imports.  As a user, I'd be a bit annoyed if I was directly pointed at an index with imports rather than to something I could actionably and immediately use.

My vote -- we keep both and let the users ctrl+click all of the links into separate tabs for fast exploration.

**Comment by TechAkayy** [comment]
Thanks @anonmos. Let's wait for @userquin's final review & closure.

**Review by TechAkayy — APPROVED** [review]

**Review by userquin — APPROVED** [review]
thx

**Pull request merged**
